keyboards/lovelive9/config.h

```diff
@@ -0,0 +1,63 @@
+
+
+#ifndef CONFIG_H
+#define CONFIG_H
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0007
+#define DEVICE_VER         0x0001
+#define MANUFACTURER    numazu Keyboards
+#define PRODUCT         lovelive9
+#define DESCRIPTION     lovelive sunshine Keyboards 9Key // Charleston, SC Meetup Handout
+
+/* key matrix size */
+#define RGBLIGHT_ANIMATIONS
+#define BACKLIGHT_LEVELS 5
+#define RGBLIGHT_EFFECT_SNAKE_LENGTH 4
+#define BREATHING_PERIOD 2
+#define RGBLIGHT_LIMIT_VAL 255
+#define RGBLIGHT_HUE_STEP 10
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+
+#define MATRIX_ROWS 9
+#define MATRIX_COLS 1
```

Comment:
Your matrix down below forgot to define the column.

Reply:
If this is like the BDN9, it would be good to align it so we can easily identify it later
```diff
-#define MATRIX_COL_PINS { }
+#define MATRIX_COL_PINS { NO_PIN }
```

Reply:
think you review and I fix config.h.
This keyboard unused diode.
All COL_PINS connect GND.